src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -55,7 +54,14 @@ protected AbstractPlatformProxy(ILoggerAdapter logger)
         private IOtelInstrumentation InternalGetOtelInstrumentation()
         {
 #if SUPPORTS_OTEL
-            return new OtelInstrumentation();
+            try
+            {
+                return new OtelInstrumentation();
+            } catch (Exception ex)

```

Comment:
Is there a more specific exception that should be caught?

Reply:
FileNotFoundException.

Reply:
I'd start with that. We can always add more types.